### PR TITLE
babl: 0.1.106 → 0.1.108

### PR DESCRIPTION
**Pull request by 9p4**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

@jtojnar

### CODE REVIEW

**Comment by amaxine** [comment]
There's a typo in the commit and PR name, s/gegl/babl/.

File: pkgs/development/libraries/babl/default.nix
Comment (jtojnar) [inline pkgs/development/libraries/babl/default.nix:1]: https://gitlab.gnome.org/GNOME/babl/-/commit/692ece4df21bf63de3b0277b4b8b97e76fff280d added `girdir` to `pkg-config` file but we are moving the `gir`s to dev output so we will need to patch the build script similarly to https://github.com/NixOS/nixpkgs/blob/93fa760066046491264a5e6fb8c82677f7b83b2f/pkgs/development/libraries/glib/split-dev-programs.patch#L35

**Review by jtojnar — CHANGES_REQUESTED** [review]
Looking at https://gitlab.gnome.org/GNOME/babl/-/compare/BABL_0_1_106...BABL_0_1_108 I noticed an issue

File: pkgs/development/libraries/babl/default.nix
Comment (9p4) [reply pkgs/development/libraries/babl/default.nix:1]: To clarify, we are moving `lib/girepository-1.0` to the dev directory?

Comment (jtojnar) [reply pkgs/development/libraries/babl/default.nix:1]: No, we are moving `share/gir-1.0` to `$dev`. So `${datadir}/gir-1.0` needs to be replaced.

Comment (9p4) [reply pkgs/development/libraries/babl/default.nix:1]: It seems like the dev package already contains `share/gir-1.0`. Am I misunderstanding what is needed to be changed?

Comment (jtojnar) [reply pkgs/development/libraries/babl/default.nix:1]: The content of the pkg-config file points to the old location here: https://gitlab.gnome.org/GNOME/babl/-/commit/692ece4df21bf63de3b0277b4b8b97e76fff280d#0cc1139e3347f573ae1feee5b73dbc8a8a21fcfa_523_526

Comment (9p4) [reply pkgs/development/libraries/babl/default.nix:1]: I see now. I'll put together a patch.

Comment (jtojnar) [reply pkgs/development/libraries/babl/default.nix:1]: I have pushed a patch fixing this.

**Review by jtojnar — APPROVED** [review]

**Pull request merged**